stable/prometheus-operator/Chart.yaml

```diff
@@ -10,7 +10,7 @@ name: prometheus-operator
 sources:
   - https://github.com/coreos/prometheus-operator
   - https://coreos.com/operators/prometheus
-version: 6.14.1
+version: 6.14.2
```

Comment:
[According to semver, since this is an enhancement, the minor version should be increased](https://semver.org/#spec-item-7)